examples/synth.rs

```diff
@@ -1,14 +1,10 @@
 //! An example of using dsp-chain's `Graph` type to create a simple Synthesiser with 3 sine wave
 //! oscillators.
 
-extern crate dsp;
-extern crate portaudio;
-
-use dsp::{Graph, Node, Frame, FromSample, Sample, Walker};
-use dsp::sample::ToFrameSliceMut;
+use sample::ToFrameSliceMut;
```

Comment:
I'd find it cleaner to leave the `sample` dependency re-exported, not just to avoid versioning conflicts, but also to provide a clean public API that does not rely on transitive dependencies.

Reply:
@fwcd Done. Can you review please?

Reply:
Looks good, I do not have write-access to this repo though, so I can‘t merge it.